plugins/modules/cloud/misc/proxmox.py

```diff
@@ -317,6 +321,18 @@
     ostemplate: local:vztmpl/ubuntu-14.04-x86_64.tar.gz'
     cores: 2
 
+- name: Create a new container with nesting enabled
+  community.general.proxmox:
+    vmid: 100
+    node: uk-mc02
+    api_user: root@pam
+    api_password: 1q2w3e
+    api_host: node1
+    password: 123456
+    hostname: example.org
+    ostemplate: local:vztmpl/ubuntu-14.04-x86_64.tar.gz'
+    features: nesting=1
```

Comment:
How about an example with more than one feature?

Reply:
I updated the example and code, how does it look?